chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductXmlPresenter.java

```diff
@@ -23,24 +28,35 @@
 
     @Override
     public int presentAsFile(final File file, final CertifiedProductDownloadResponse cpList) {
-        int numRecords = 0;
-        FileOutputStream os = null;
         try {
-            os = new FileOutputStream(file);
-            Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
-            marshaller.setClassesToBeBound(cpList.getClass());
-            marshaller.marshal(cpList, new StreamResult(os));
-            numRecords = (cpList.getListings() == null ? 0 : cpList.getListings().size());
-        } catch (final FileNotFoundException ex) {
-            LOGGER.error("file not found " + file);
-        } finally {
-            if (os != null) {
-                try {
-                    os.close();
-                } catch (final IOException ignore) {
+            Writer writer = null;
+            XMLStreamWriter streamWriter = null;
+            try {
+                XMLOutputFactory factory = XMLOutputFactory.newInstance();
+                writer = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
+                //streamWriter = factory.createXMLStreamWriter(writer);
+                streamWriter = new IndentingXMLStreamWriter(factory.createXMLStreamWriter(writer));
+                streamWriter.writeStartDocument("UTF-8", "1.0");
+                streamWriter.writeStartElement("ns2:results");
+                streamWriter.writeNamespace("ns2", "http://chpl.healthit.gov/listings");
```

Comment:
Should this use the `environment.get(urlStart)` for the namespace? Also, what's the namespace? I don't think the URL here resolves to anywhere; does it need to?

Reply:
The enviroment.get thing will get something with https. If the original file had http then we should probably keep it there

Reply:
It's just a made up URL to use for the namespace.

Reply:
Looking at the chplUrlBegin parameter, I'm not sure it would be correct...  But the URL is plastered all over the JAXB annotations, so I think we should just leave it alone.